sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetTypeWideningSuite.scala

```diff
@@ -166,11 +166,17 @@ class ParquetTypeWideningSuite
       (Seq("1", "2", Int.MinValue.toString), LongType, IntegerType),
       (Seq("1.23", "10.34"), DoubleType, FloatType),
       (Seq("1.23", "10.34"), FloatType, LongType),
-      (Seq("1.23", "10.34"), LongType, DateType),
-      (Seq("1.23", "10.34"), IntegerType, TimestampType),
-      (Seq("1.23", "10.34"), IntegerType, TimestampNTZType),
       (Seq("2020-01-01", "2020-01-02", "1312-02-27"), DateType, TimestampType)
-    )
+    ) ++ {
+      if (SQLConf.get.ansiEnabled) Nil
+      else {
+        Seq(
+          (Seq("1.23", "10.34"), LongType, DateType),
```

Comment:
shall we just use valid int values?

Reply:
should be ok

Reply:
[d95098c](https://github.com/apache/spark/pull/44481/commits/d95098c75bce2881913cb0efb8ab518efd9db372) change to use `1` and `10` as input